### PR TITLE
updates UI to new coaching state

### PR DESCRIPTION
**Pull request by anene**

### CODE REVIEW

File: css/live.css [hunk 1]
```diff
@@ -27,11 +25,25 @@
     margin-top: 10px;
 }
 
+.panel-body {
+    padding: 6px 12px;
+    width: 250px;
+    min-width: 250px;
+    max-width: 250px;
```
Comment (cbonnoit) [inline css/live.css]: is this the same as `width: 250px`?

File: pages/live/index.html [hunk 2]
```diff
@@ -5,75 +5,92 @@
...
     <script src="index.js", type="module"></script>
 </head>
 
-<body>
-    <input type="checkbox" id="realtimeEnabled" name ="realtimeEnabled">
-    <label for="realtimeEnabled">Enabled</label>
-    <div id="session"></div>
+<body class="dark-mode">
+    <!-- <input type="checkbox" id="realtimeEnabled" name ="realtimeEnabled">
+    <label for="realtimeEnabled">Enabled</label> -->
```
Comment (cbonnoit) [inline pages/live/index.html]: i think we can delete this commented out code

File: pages/live/index.js [hunk 3]
```diff
@@ -173,13 +205,26 @@ function _extendLastTranscript (text) {
...
-  const nameContentNode = transcriptRoot.appendChild(createNode('div', {'class': 'transcriptNameContent'}))
+
+  const transcriptRoot = root.appendChild(createNode('div', {'class': 'flexColumn transcript'}))
+
+  let transcriptHeader
+  let contentStyle
+
+  if (partyCode == 0) {// prospect and should be RHS
+    transcriptHeader = transcriptRoot.appendChild(createNode('div', {'class': 'flexRow', 'style': 'margin-left: 20%'}))
+    contentStyle = 'margin-left: 20%'
```
Comment (cbonnoit) [inline pages/live/index.js:216]: i would have expected that the easiest way to align something to the rhs would be with something like `class: 'flexRowRight'`, but i'm fine with this if it looks like what you want it to look like

**Review by cbonnoit — APPROVED** [review]

File: pages/live/index.js (hunk 3 above)
Comment (anene) [reply pages/live/index.js:216]: didn't want to align it to the right, want it to be floating a bit to the left

**Pull request merged**